lib/tapioca/dsl/helpers/active_record_column_type_helper.rb

```diff
@@ -108,6 +116,16 @@ def lookup_arg_type_of_method(column_type, method)
 
           first_argument_type.to_s
         end
+
+        sig { params(column_type: ActiveRecord::Enum::EnumType).returns(String) }
+        def enum_setter_type(column_type)
+          case column_type.subtype
```

Comment:
This breaks rails < 7 because `subtype` was previously private. Rails commit: https://github.com/rails/rails/commit/70259f5e519869052a25db2c832e2c9d53349c66#diff-a0de22dc0114613fdc67f48c916e70b9983ef47bcc84b96232c36398af1d166fR156-R159